client/core/trade.go

```diff
@@ -605,8 +607,10 @@ func (t *trackedTrade) matchIsActive(match *matchTracker) bool {
 		// - MakerSwapCast requires no further action from the taker
 		// - (TakerSwapCast requires action on both sides)
 		// - MakerRedeemed requires no further action from the maker
+		// - MatchComplete requires no further action. This happens if taker
+		//   does not have server's ack of their redeem request (RedeemSig).
 		status, side := match.MetaData.Status, match.Match.Side
-		if status == order.NewlyMatched ||
+		if status == order.NewlyMatched || status == order.MatchComplete ||
```

Comment:
In testing by blocking the taker's `'redeem'` request to get a revoke with the server at `MakerRedeemed`, I realized that because the `RedeemSig` is absent (`match.MetaData.Status == order.MatchComplete && len(proof.Auth.RedeemSig) > 0` no longer hits), the match would never be retired even though it was revoked and MatchComplete.